vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -511,6 +499,53 @@ func resourceVSphereVirtualMachineUpdate(d *schema.ResourceData, meta interface{
 		return err
 	}
 
+        // Read the VM properties (memory and cpu hot add properties)
+        var mov mo.VirtualMachine
```

Comment:
Remove the tab expansion to spaces and keep the tables.
go fmt <filename> and check-in